record/record.app.js

```diff
@@ -147,6 +148,17 @@
                                 selectable: false
                             };
                             $rootScope.tableModels[i] = model;
+
+                            if($rootScope.tableModels[i].rowValues.length == 0){
+                              $rootScope.resultCountString[i]='(no results found)';
+                            }
+                            else if($rootScope.tableModels[i].hasNext){
+                              $rootScope.resultCountString[i]='(showing first '+ $rootScope.tableModels[i].rowValues.length +' results)';
```

Comment:
Instead of creating a new array and attaching it to the scope, add this string to the existing table model.
`model.resultCountString = "showing first..."`